pom.xml

```diff
@@ -1,14 +1,15 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>org.eenterphace.mbechler</groupId>
+	<groupId>com.gresham</groupId>
 	<artifactId>marshalsec</artifactId>
-	<version>0.0.3-SNAPSHOT</version>
+	<version>1.0.0.RELEASE</version>
```

Comment:
Something to consider, we could version it similarly/the same as the original so we know what version we are replicating e.g. 0.0.3.RELEASE. The fact it has a different groupId should be enough to suggest it isn't the original artifact from mbechler.

What happens if we make a mistake or need to build again for the same version 0.0.3.RELEASE, I hear you ask? That is when can append an additional digit to the version.

I'm basing this on my observations of birt and openid connect, I haven't had to build these myself so take all of the above with a pinch of salt :)

Reply:
I can vouch for open id connect - we base our version of v1.3.3 - our versioning is 1.3.3.GRESHAM-<patchVersion> so currently we are 1.3.3.GRESHAM-21 and could quite easily pull in upstream changes of 1.3.3 into our branch.  We increment the patch version for each release.  

Birt follows something similar - although it is vile and gets worse with each iteration - we use v4.5.0 from eclipse and our current version is 4.5.0a-9.2.  We increment the digit in the position of the 9 or the 2 with each change depending on how drastic the change is - sort of semantic versioning style but with a version that does not obey semantic versioning